backend/reviving_ontology_api_stuff.py

```diff
@@ -57,55 +46,6 @@
     allow_headers=['*']
 )
 
-# get actionType endpoints:
-# curl -H "Content-type: application/json" -H "Authorization: Bearer $PALANTIR_ENCLAVE_AUTHENTICATION_BEARER_TOKEN" "https://unite.nih.gov/api/v1/ontologies/ri.ontology.main.ontology.00000000-0000-0000-0000-000000000000/actionTypes"
-
-createNewConceptSetDescription = {
```

Comment:
I deleted some redundant comments from `reviving_ontology_api_stuff.py`, and moved other bits into `new_enclave_api.py` (which will be renamed when I'm done w/ it)